benchmarks/dotnet-format/perf/dotnet-format.PerformanceTests.csproj

```diff
@@ -0,0 +1,26 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
I created a new directory here `benchmarks` here to put the benchmark project. (can't be put under tests folder because it has specified test config)